### PR TITLE
Do it task view

### PR DESCRIPTION
**Pull request by rmisutyte**
### ISSUE/TICKET NUMBER
104 - Create an initial Do It task view

### Describe the changes
- Added task anxiety score view where user is able to enter anxiety scores for the task
- Removed AnxietyScore model and moved the score choices inside AnxietyScoreCard model

### Screenshots (if appropriate)
screenshot-127 0 0 1-8000-2017-04-26-22-36-35

### Questions or comments (if any)

### CODE REVIEW

File: ocdaction/ocdaction/templates/dashboard/act/task_list.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
                         <li><p>Goals: My task goals are {{ task.task_goals }}<p></li>
                         <li><p>Compulsions: My task compulsions are {{ task.task_compulsions }}<p></li>
                     </ul>
+                    <a href="{% url 'task-score-form' task.id %}"><p> Do the task </p></a>
```
Comment (lili2311) [inline ocdaction/ocdaction/templates/dashboard/act/task_list.html]: a is a text field so no need to have the p tags inside as well.

File: ocdaction/tasks/models.py [hunk 2]
```diff
@@ -42,79 +42,54 @@ class AnxietyScore(models.Model):
...
-    score_after_60_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_60',
-        on_delete=models.CASCADE
+    score_after_60_min = models.CharField(
+        max_length=2,
+        choices=ANXIETY_SCORE_CHOICES,
+        default=SCORE_ZERO
```
Comment (lili2311) [inline ocdaction/tasks/models.py]: I feel like 0 shouldn't be a default as it is the best score you can have aka No Anxiety, perhaps a 5 or even blank would be better so we can always tell when a user has not entered the score. In fact blank is best

Comment (rmisutyte) [inline ocdaction/tasks/models.py]: I have removed the default choice 0, so now default is blank

**Comment by rmisutyte** [comment]
@lili2311 , I have updated pull request according to the requested changes, can you have a look when you will have a minute? Thank you :)

**Comment by lmash** [comment]
I've reviewed final changes and run the changes (plus migrations) on my side.. all looks good

**Comment by mybits** [comment]
Thanks @rmisutyte for PR and @lili2311  @lmash for reviewing. Merging :shipit:

**Comment by mybits** [comment]
Hi @rmisutyte! It would be great to have it merged 👍 are you around for resolving the conflicts? 🥇

**Pull request merged**